zanata-war/src/main/java/org/zanata/rest/editor/MediaTypes.java

```diff
@@ -32,6 +32,11 @@ public String toString() {
     public static final String APPLICATION_ZANATA_LOCALES_JSON =
             APPLICATION_ZANATA_LOCALES + JSON;
 
+    public static final String APPLICATION_ZANATA_SUGGESTIONS =
+            APPLICATION_VND_ZANATA + ".suggestions";
+    public static final String APPLICATION_ZANATA_SUGGESTIONS_JSON =
+            APPLICATION_ZANATA_SUGGESTIONS + JSON;
+
```

Comment:
I do not see a clear course of action here, and it seems that at the moment it would cause no problems to keep or remove the vnd mime type. Is everyone happy with leaving this as-is for now to keep it consistent with the adjacent media type strings?

- [ ] @huangp 
- [ ] @carlosmunoz 
- [ ] @seanf